components/Search/SearchForm.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import ActionSearch from 'material-ui/svg-icons/action/search';
+
+class SearchForm extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { hintText, onSubmit, query } = this.props;
+
+    const onEnter = (e) => {
+      
+      if (e.keyCode !== 13)
```

Comment:
brackets, also @mike-robertson is this the recommended way to do search forms in react?